modules/custom/az_core/az_core.services.yml

```
@@ -25,12 +25,13 @@ services:
       - '@config_update.config_list'
       - '@module_handler'
   # Decorates the normal user module links.
-  az_core.toolbar_link_builder:
+  az_person.toolbar_link_builder:
```

Does this need to change back?
```suggestion
  az_core.toolbar_link_builder:
```